### PR TITLE
add info about iris

### PR DESCRIPTION
**Pull request by topialla**
as promised, a few sentences about IRIS connect

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # WhoWeAre
-Documents about Innovationsverbund Öffentliche Gesundheit (InÖG)
+Documents about Innovationsverbund Öffentliche Gesundheit (InÖG) (loosely translated: Public healt innovation network).
```
Comment (bkastl) [inline README.md]: Public healt**h** innovation network

File: README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
 ## What one can find in here
 
-Here we put documents about the Innovationsverbund Öffentliche Gesundheit. There is a lot of German involved, as we are an association based in Germany. We are currently working on English versions. Our statutes will always be in German due to legal reasons.
+Here we put documents about and projects the Innovationsverbund Öffentliche Gesundheit (InÖG).
```
Comment (bkastl) [inline README.md]: Here we put documents about the Innovationsverbund Öffentliche Gesundheit (InÖG) and share our projects?

**Review by bkastl — CHANGES_REQUESTED** [review]
Minor Tweaks required

File: README.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+
+Our first and most mature project is [https://iris-connect.de](IRIS connect) ([https://github.com/iris-connect/](GitHub)), the e2e-encrypted data interface in and between public health agencies.
+The first use case of this base architecture is transfer of contact tracing data, more is planned and specified at the moment. If you're looking for a place to contribute, this would be a good project to start with.
```
Comment (LittleDetritus) [inline README.md]: IRIS is currently operational in the following german states: North Rhine-Westphalia, Saxony, Thuringia, Hesse and of late parts of Berlin.

**Pull request merged**